library/build.gradle

```diff
@@ -39,4 +40,10 @@ publish {
     publishVersion = baristaVersion
     desc = 'The guy who serves a great Espresso'
     website = 'https://github.com/SchibstedSpain/Barista'
+}
+
+// Exclude Kotlin files from Javadoc.
```

Comment:
Why we need it? The comment explain what it does, but not why. At the first look, it sounds good to include the Kotlin files to Javadoc, isn't it?

Reply:
It's explained in the link on the next line

Reply:
This answer smashed my proactivity. As you have guessed, I noticed the link on the next line. Adding a three or four words to sum up the reason of adding this patch will save some time to other collaborators of reading the whole GitHub issue and lose their task focus.

Reply:
The only reason we could write is "it doesn't work", and I don't see how that would be helpful.
I'm sure your proactivity can forgive this one :D

Maybe it would have been clearer if done in a separate PR, to clarify it's a separate issue from the topic of this PR.